chacha20poly1305/src/lib.rs

```diff
@@ -26,7 +29,35 @@ pub enum AeadError {
     InvalidCiphertext,
 }
 
+impl alloc::fmt::Display for AeadError {
```

Comment:
```suggestion
impl core::fmt::Display for AeadError {
```
I think that also works? Then you don't need to pull in `alloc` at all.

Reply:
huh, TIL!